Description: support reading of default value for startDragDistance property
from QT_USE_DRAG_DISTANCE environment variable.
See also http://bugreports.qt.nokia.com/browse/QTBUG-12594
Author: Joonas Tanskanen <joonas.tanskanen@sasken.com>
---
 src/gui/kernel/qapplication.cpp |    5 +++++
 1 file changed, 5 insertions(+)

--- a/src/gui/kernel/qapplication.cpp
+++ b/src/gui/kernel/qapplication.cpp
@@ -1027,6 +1027,11 @@ void QApplicationPrivate::initialize()
     q->setAttribute(Qt::AA_S60DisablePartialScreenInputMode);
 #endif
 
+    if(qgetenv("QT_USE_DRAG_DISTANCE").toInt() > 0) {
+        int dragThreshold = qgetenv("QT_USE_DRAG_DISTANCE").toInt();
+        q->setStartDragDistance(dragThreshold);
+    }
+
     if (qt_is_gui_used)
         initializeMultitouch();
 }
